slash/loader.py

```diff
@@ -51,10 +53,12 @@ def _get_matchers(self):
 
     def get_runnables(self, paths):
         assert context.session is not None
-
         sources = (t for repetition in range(config.root.run.repeat_all)
                    for t in self._generate_test_sources(paths))
         returned = self._collect(sources)
+        self._duplicate_funcs |= self._local_config.duplicate_funcs
+        for (path, name, line) in self._duplicate_funcs:
```

Comment:
@maormarcus Perhaps it would be a good idea to use `sorted` to have the duplicates emitted in some sane order here